INSTALL.md

```diff
@@ -0,0 +1,25 @@
+
+## Setup CactusBot:
+```
+git clone https://github.com/cactusdev/cactusbot
+cd cactusbot
+pip3 install -r requirements.txt
+cp config.template.py config.py
+```
+
+Next, open `config.py` with your favorite text editor, and set 
+`USERNAME` and `PASSWORD` to the bots login info, and set `CHANNEL` to your channel's name.
+
+# Usage
+
+Start Sepal:
+
+`npm start`
```

Comment:
And we should be assuming that this is a 100% fully hosted instance. We shouldn't assume that they want to use ours because in the future the only way to make an account will be though the panel, and then you have a bot anyways